RaveBusinessLogic/VBET.xml

```diff
@@ -1,145 +1,50 @@
-      <Node label="Inputs">
-        <Children collapsed="true">
-          <Node label="Hydrography">
-            <Children collapsed="true">
-              <Node xpathlabel="Name" xpath="Intermediates/Geopackage/Layers/Vector[@id='VBET_NETWORK']" type="line" symbology="vbet_network" />
-              <Node label="VBET Network with Bankfull" xpath="Intermediates/Geopackage/Layers/Vector[@id='VBET_NETWORK']" type="line" id="network" symbology="bankfull_flowlines" />
-              <Node label="Full NHD HR+ Drainage Network" xpath="Inputs/Geopackage/Layers/Vector[@id='FLOWLINES']" type="line" symbology="flow_lines" />
-              <Node label="Perennial Drainage Network" xpath="Inputs/Geopackage/Layers/Vector[@id='FLOWLINES']" type="line" symbology="nhdperrenial" />
-              <Node label="Intermittent Drainage Network" xpath="Inputs/Geopackage/Layers/Vector[@id='FLOWLINES']" type="line" symbology="nhdintermittent" />
-              <Node label="Ephemeral Drainage Network" xpath="Inputs/Geopackage/Layers/Vector[@id='FLOWLINES']" type="line" symbology="nhdephemeral" />
-              <Node label="Network - Upstream Drainage Network" xpath="Inputs/Geopackage/Layers/Vector[@id='FLOWLINES']" type="polygon" symbology="upstreamdrainagearea" />
-              <Node label="River Polygons (NHD Area)" xpath="Inputs/Geopackage/Layers/Vector[@id='FLOW_AREA']" type="polygon" symbology="nhdarea" />
-              <Node xpathlabel="Name" xpath="Inputs/Geopackage/Layers/Vector[@id='CATCHMENTS']" type="polygon" symbology="" />
-              <Node xpathlabel="Name" xpath="Intermediates/Raster[@id='CHANNEL_BUFFER_RASTER']" type="raster" symbology="" />
-              <Node xpathlabel="Name" xpath="Intermediates/Raster[@id='FLOW_AREA_RASTER']" type="raster" symbology="" />
-            </Children>
-          </Node>
-          <Node label="Topography">
-            <Children collapsed='true'>
-              <Node label="Slope Analysis" xpath="Inputs/Raster[@id='SLOPE_RASTER']" type="raster" symbology="slope" transparency="40" />
-              <Node label="Height Above Nearest Drainage" xpath="Intermediates/Raster[@id='HAND_RASTER']" type="raster" symbology="hand" transparency="40" />
-              <Node label="Digital Elevation Model" xpath="Inputs/Raster[@id='DEM']" type="raster" symbology="dem" transparency="40" id="dem" />
-              <Node xpathlabel="Name" xpath="Inputs/Raster[@id='HILLSHADE']" type="raster" symbology="" id="hillshade" />
+              <Node label="Estimated Active Channel" xpath="Geopackage/Layers/Vector[@id='VBET_100']" type="polygon" symbology="vbet100outline" id="vbet100outline"/>
+              <Node label="Estimated Active Floodplain" xpath="Geopackage/Layers/Vector[@id='VBET_70']" type="polygon" symbology="vbet70outline" id="vbet70outline"/>
+              <Node label="Estimated Inactive Floodplain" xpath="Geopackage/Layers/Vector[@id='VBET_50']" type="polygon" symbology="vbet50outline" id="vbet50outline"/>      

```

Comment:
So you can now see in the `project.rs.xml' on https://github.com/Riverscapes/RiverscapesData/issues/19 what these will eventually become:
![image](https://user-images.githubusercontent.com/9723905/129596903-819430fa-6217-47ad-bdf5-13065673bdbd.png)

These are 'thresh_68' for floodplain and 'thresh_80 for inactive. So replace from VBET_70 and VBET_50 accordingly.